tests/compilation/test_predictor_rl.py

```diff
@@ -53,9 +55,7 @@ def test_qcompile_with_newly_trained_models() -> None:
         test=True,
     )
 
-    res = rl.qcompile(qc, figure_of_merit=figure_of_merit, device_name=device)
-    assert isinstance(res, tuple)
-    qc_compiled, compilation_information = res
+    qc_compiled, compilation_information = rl.qcompile(qc, figure_of_merit=figure_of_merit, device_name=device)
```

Comment:
At some point it would really be great to add some more meaningful tests that actually test something.
What comes to mind is running, e.g. QCEC on the resulting circuits to ensure that they are equivalent.